### PR TITLE
[🐴] List Adjustments

### PR DESCRIPTION
**Pull request by haileyok**
- Prevent `scrollToEnd` from being called if a scroll is already in progress
- Animate the view up while keyboard is opening

### CODE REVIEW

File: src/screens/Messages/Conversation/MessagesList.tsx [hunk 1]
```diff
@@ -222,15 +248,3 @@ export function MessagesList() {
...
 }
-
-function useKeyboardVerticalOffset() {
-  const {top: topInset} = useSafeAreaInsets()
-
-  return Platform.select({
-    ios: topInset,
-    // I thought this might be the navigation bar height, but not sure
-    // 25 is just trial and error
-    android: 25,
```
Comment (haileyok) [inline src/screens/Messages/Conversation/MessagesList.tsx:233]: @mozzius This `25` actually was giving me too much space 🤔 seems like zero is better?

**Comment by haileyok** [comment]
@mozzius by no means is this necessarily the best way to do this! I was just messing around with it and got it working pretty smoothly but if there's a better way we can totally do that.

File: src/screens/Messages/Conversation/MessagesList.tsx (hunk 1 above)
Comment (mozzius) [reply src/screens/Messages/Conversation/MessagesList.tsx:233]: Hmmm what on earth is that value supposed to be then. `25` is definitely right for my Pixel 3a emulator

**Review by mozzius — APPROVED** [review]
Looks good, I'll carry on investigating `keyboardVerticalOffset` next week in a different PR

**Comment by mozzius** [comment]
> @mozzius by no means is this necessarily the best way to do this! I was just messing around with it and got it working pretty smoothly but if there's a better way we can totally do that.

If you got it working then it should be super simple to add interactive dismissal, nice work!

**Pull request merged**